### PR TITLE
WEBAPP-512: Fix (link) accessibility

### PR DESCRIPTION
**Pull request by klinzo**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/). 
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **WEBAPP**.

Also fixes
- https://issues.integreat-app.de/browse/WEBAPP-397 apart from the fact that the dropDown is not closable with a press of `Enter` from outside the dropdown. I created an issue for this: https://issues.integreat-app.de/browse/WEBAPP-514
- https://issues.integreat-app.de/browse/WEBAPP-396 (only partly, everything else is still selectable). Will create a separate branch and PR for this.

### CODE REVIEW

File: src/modules/layout/components/__tests__/HeaderDropDown.spec.js [hunk 1]
```diff
@@ -62,10 +62,10 @@ describe('HeaderDropDown', () => {
     })
   })
 
-  it('should toggle when user clicks on img', () => {
+  it('should toggle when user clicks on button', () => {
     const component = wrapperComponent.dive()
     const instance: any = component.instance()
-    const onClick = component.find('img').prop('onClick')
+    const onClick = component.find({ selector: 'button' }).prop('onClick')
```
Comment (klinzo) [inline src/modules/layout/components/__tests__/HeaderDropDown.spec.js]: Any better way to do this?

Comment (maxammann) [reply src/modules/layout/components/__tests__/HeaderDropDown.spec.js]: I think this is fine

File: src/modules/layout/components/LocationFooter.js [hunk 2]
```diff
@@ -21,8 +21,10 @@ export class LocationFooter extends React.PureComponent<PropsType> {
     const { t, city, language, onClick } = this.props
 
     return <Footer onClick={onClick}>
-      <CleanLink to={new DisclaimerRouteConfig().getRoutePath({ city, language })}>{t('imprintAndContact')}</CleanLink>
-      <CleanAnchor href='https://integreat-app.de/datenschutz/'>{t('privacy')}</CleanAnchor>
+      <CleanLink aria-label={t('imprintAndContact')} to={new DisclaimerRouteConfig().getRoutePath({ city, language })}>
```
Comment (Schedulaar) [inline src/modules/layout/components/LocationFooter.js]: Also here, an `aria-label` is not necessary.

File: src/modules/layout/components/LocationFooter.js [hunk 3]
```diff
@@ -21,8 +21,10 @@ export class LocationFooter extends React.PureComponent<PropsType> {
...
+        {t('imprintAndContact')}
+      </CleanLink>
+      <CleanAnchor title={t('privacy')} href='https://integreat-app.de/datenschutz/'>{t('privacy')}</CleanAnchor>
```
Comment (Schedulaar) [inline src/modules/layout/components/LocationFooter.js]: A `title` is not appropriate here. Just leave it as is.

File: src/modules/layout/components/ToolbarItem.js [hunk 4]
```diff
@@ -20,7 +20,7 @@ class ToolbarItem extends React.PureComponent<PropsType> {
   render () {
     const { href, text, icon } = this.props
     return (
-      <StyledToolbarItem href={href} aria-label={text}>
+      <StyledToolbarItem href={href} title={text}>
```
Comment (Schedulaar) [inline src/modules/layout/components/ToolbarItem.js]: `aria-label` was better than `title`. Title doesn't make sense here. (We already have a custom tooltip for that)

File: src/modules/common/components/Tile.js [hunk 5]
```diff
@@ -82,7 +82,7 @@ class Tile extends React.PureComponent<PropsType> {
     if (!tile.isExternalUrl) {
       return <CleanLink to={tile.path}>{this.getTileContent()}</CleanLink>
     } else if (!tile.postData) {
-      return <CleanAnchor href={tile.path}>{this.getTileContent()}</CleanAnchor>
+      return <CleanAnchor title={tile.title} href={tile.path}>{this.getTileContent()}</CleanAnchor>
```
[the remark this answers is not in the document]
Comment (klinzo) [reply src/modules/common/components/Tile.js]: Actually, I don't know anymore why I did what I did here. It did make sense to me tough. But yeah, I guess it's totally redundant.

**Pull request merged**